pom.xml

```diff
@@ -47,29 +48,40 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <functional.java>4.5</functional.java>
+        <java.level>6</java.level>
```

Comment:
Why java 6?

We should be building/supporting Java 8.  Even if we're not using Java 8 syntax at this time.